internal/testing/exchange/exchange.go

```diff
@@ -103,29 +106,38 @@ func MockWSInstance[T any, PT interface {
 	}
 
 	b.Features.Subscriptions = []*subscription.Subscription{}
+	// Exchanges which don't support subscription conf; Can be removed when all exchanges support sub conf
+	b.Websocket.GenerateSubs = func() ([]subscription.Subscription, error) { return []subscription.Subscription{}, nil }
```

Comment:
Carry-over from something else? kraken and binance both have it set. Does not seem relevant

Reply:
We're disabling default subscriptions for both config sub exchanges and for hardcoded ones.
If we don't do this then any mock server would have to also handle the subscriptions.
And obviously GenerateSubscriptions and WsSubscribe are unit tested separately anyway.

If you mean that not both these methods of disabling the Subs are needed as it stands, then I definitely want to keep them, because this MockWsInstance should be available to work for any exchange immediately and not bear-trap any new users.

Does this help? 8cc728fd

Reply:
Sure! I don't think I absorbed what was going on